server/controllers/stock/functions/rumer.function.js

```diff
@@ -0,0 +1,192 @@
+const _ = require('lodash');
+const moment = require('moment');
+
+const db = require('../../../lib/db');
+const core = require('../core');
+
+exports.getData = getData;
+
+// default report parameters
+const DEFAULT_PARAMS = {
+  csvKey : 'rumer',
+  filename : 'REPORT.RUMER.TITLE',
+  orientation : 'landscape',
+};
+
+/**
+ * @method getRumer
+ *
+ * @description
+ * This method builds the RUMER (Registre d’Utilisation des Médicaments Et Recettes) report by month
+ *
+ * GET /api/stock/rumer
+ */
+async function getData(reqQuery) {
+  const params = reqQuery;
+  params.exclude_out_stock = parseInt(params.exclude_out_stock, 10);
+  params.include_daily_balances = parseInt(params.include_daily_balances, 10);
+  params.condensed_report = parseInt(params.condensed_report, 10);
+
+  const data = {};
+  const headerReport = [];
+  const configurationData = [];
+
+  params.start_date = moment(new Date(params.start_date)).format('YYYY-MM-DD');
+  params.end_date = moment(new Date(params.end_date)).format('YYYY-MM-DD');
+
+  const startDate = parseInt(moment(params.start_date).format('DD'), 10);
```

Comment:
This is a bit complicated to read.  I propose you use `moment.diff()` to make it a bit simpler since you are using moment anyway.  Something like:

```js
const numDays = moment(params.end_date).diff(params.start_date, 'days');
for (let i = 0; i <= numDays; i++) {
  headerReport.push(i);
}
```

Or something similar (you might have to add or subtract 1 from `i`).